build/raster-cli/raster-cli/raster-cli.vcxproj

```
@@ -89,6 +89,7 @@
       <Optimization>Disabled</Optimization>
       <SDLCheck>true</SDLCheck>
       <ConformanceMode>true</ConformanceMode>
+      <AdditionalIncludeDirectories>C:\Users\Yoanna\Desktop\raster-cli\src\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Please, use relative paths.

---

Also in line 73 I apparently made the same mistake... oops :) Can you please fix it in this change?

---

```suggestion
<AdditionalIncludeDirectories>C:\Users\Yoanna\Desktop\raster-cli\src\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```
If it is more comfortable code style for you, we may use PascalCase for dir and project names also? I'm totally O.K. with any case. Feel free to disregard this suggestion.

---

In my opinion, It will be easy if the class and their files have the same names.